themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Improving the Pulumi Service Experience"
```

Comment:
i still think the title can be improved. it sounds a little generic and i think it understates the work you've done.

Reply:
perhaps hinting at the nature of the improvements might make this more catchy to a reader. i don't think there's anything wrong with "improving" inherently, but i am feeling like the title is non-specific and doesn't have enough draw

Reply:
Yeah - let's do something more exciting here :-).

"A New and Improved User Experience For the Pulumi Service"

"A New Pulumi Service UI: Rich Dashboards, Easier Navigation and Favorites"

(or something else - but maybe more in those directions?)